parm/anlstat/anlstat_base_config.yaml.j2

```diff
@@ -21,10 +21,18 @@ atmos:
   stat_file_path: '{{ COMIN_ATMOS_ANALYSIS }}'
   stat_file_name: 'stat.atm.tar'
   obs spaces:
+  - name: prepbufr_adpsfc
```

Comment:
I think the naming convention here is changing, look at https://github.com/NOAA-EMC/GDASApp/blob/b750ddac4ec4ef1b48174fe76e53db2c9bad3be9/parm/anlstat/anlstat_base_config.yaml.j2#L31-L49